core/store/src/main/java/uk/gov/gchq/gaffer/store/operation/handler/GetVariablesHandler.java

```diff
@@ -22,13 +22,18 @@
 import uk.gov.gchq.gaffer.store.Store;
 
 import java.util.HashMap;
+import java.util.List;
 import java.util.Map;
 
 public class GetVariablesHandler implements OperationHandler<GetVariables> {
     @Override
     public Map<String, Object> doOperation(final GetVariables operation, final Context context, final Store store) throws OperationException {
         final Map<String, Object> variableMap = new HashMap<>();
-        for (final String key : operation.getVariableNames()) {
+        List<String> variableNames = operation.getVariableNames();
```

Comment:
This will probably read better just as one line:
```List<String> variableNames = operation.getVariableNames() != null ? operation.getVariableNames() : Lists.newArrayList();```
I know returning earlier will be more efficient but the iteration of an empty list should not really affect performance.

Reply:
I agree, it's negligible difference. I have updated